src/convertToObject.js

```diff
@@ -12,7 +12,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+ const newObject = {};
+
+  const filterString = sourceString.split(';');
+
+  for (let i = 0; i < filterString.length; i++) {
+    let propertyAndValue = filterString[i].trim();
+
+    if (propertyAndValue === '') {
+      continue;
+    }
+
+    propertyAndValue = propertyAndValue.split(':');
```

Comment:
Use destructuring to directly extract the key and value from each key-value pair